<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureWeb</class>
 <widget class="QWidget" name="ConfigureWeb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBoxWebConfig">
       <property name="title">
        <string>yuzu 网络服务</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutYuzuWebService">
        <item>
         <widget class="QLabel" name="web_credentials_disclaimer">
          <property name="text">
           <string>通过提供您的用户名和令牌，你同意让yuzu收集更多的使用数据，其中可能包括用户标识信息.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayoutYuzuUsername">
          <item row="2" column="3">
           <widget class="QPushButton" name="button_verify_login">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>校验</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="web_signup_link">
            <property name="text">
             <string>注册</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QLabel" name="username" />
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_token">
            <property name="text">
             <string>令牌: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_token_verified">
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_username">
            <property name="text">
             <string>用户名: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QLineEdit" name="edit_token">
            <property name="maxLength">
             <number>80</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="web_token_info_link">
            <property name="text">
             <string>什么是我的令牌？</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>遥测</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="toggle_telemetry">
          <property name="text">
           <string>与yuzu团队分享匿名使用数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="telemetry_learn_more">
          <property name="text">
           <string>学到更多</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayoutTelemetryId">
          <item row="0" column="0">
           <widget class="QLabel" name="label_telemetry_id">
            <property name="text">
             <string>遥测 ID:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="button_regenerate_telemetry_id">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>再生</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
     <widget class="QGroupBox" name="discord_group">
      <property name="title">
       <string>不和谐的存在</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_21">
       <item>
        <widget class="QCheckBox" name="toggle_discordrpc">
         <property name="text">
          <string>显示当前游戏中你的不和谐状态</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
